x-pack/plugins/infra/public/components/waffle/waffle_metric_controls.tsx

```diff
@@ -11,96 +11,189 @@ import {
   EuiFilterGroup,
   EuiPopover,
 } from '@elastic/eui';
+import { FormattedMessage, InjectedIntl, injectI18n } from '@kbn/i18n/react';
 import React from 'react';
 import { InfraMetricInput, InfraMetricType, InfraNodeType } from '../../../common/graphql/types';
 interface Props {
   nodeType: InfraNodeType;
   metric: InfraMetricInput;
   onChange: (metric: InfraMetricInput) => void;
+  intl: InjectedIntl;
 }
 
-const OPTIONS = {
-  [InfraNodeType.pod]: [
-    { text: 'CPU Usage', value: InfraMetricType.cpu },
-    { text: 'Memory Usage', value: InfraMetricType.memory },
-    { text: 'Inbound Traffic', value: InfraMetricType.rx },
-    { text: 'Outbound Traffic', value: InfraMetricType.tx },
-  ],
-  [InfraNodeType.container]: [
-    { text: 'CPU Usage', value: InfraMetricType.cpu },
-    { text: 'Memory Usage', value: InfraMetricType.memory },
-    { text: 'Inbound Traffic', value: InfraMetricType.rx },
-    { text: 'Outbound Traffic', value: InfraMetricType.tx },
-  ],
-  [InfraNodeType.host]: [
-    { text: 'CPU Usage', value: InfraMetricType.cpu },
-    { text: 'Memory Usage', value: InfraMetricType.memory },
-    { text: 'Load', value: InfraMetricType.load },
-    { text: 'Inbound Traffic', value: InfraMetricType.rx },
-    { text: 'Outbound Traffic', value: InfraMetricType.tx },
-    { text: 'Log Rate', value: InfraMetricType.logRate },
-  ],
+let OPTIONS: { [P in InfraNodeType]: Array<{ text: string; value: InfraMetricType }> };
```

Comment:
I don't think these options should be internationalized -- they directly map to the field names in the reported data, these field names will not be internationalized either. They are technical terms, and changing them will make the UI more confusing for the user.